src/views/inline_footnote.ts

```diff
@@ -55,6 +59,17 @@ export class InlineFootnoteView<
 {
   protected popperContainer: HTMLDivElement
 
+  public isSelected() {
+    const sel = this.view.state.selection
+    const juxtaposed =
```

Comment:
what does that mean?

Reply:
means it's positioned just near this node. I'll be happy to rename it if you'd like to suggest other name.

Reply:
Based on the condition` sel.$head.pos === sel.$anchor.pos && sel.$head.pos === this.getPos() + this.node.nodeSize`, it seems we are checking if the cursor is at the end of the node (isCursorAtNodeEnd). Am I right?

Reply:
that's right, we consider cursor at the end of the node to be a selection of the node, this is because that node is node selectable by prosemirror